docs/parallel_training/setup.rst

```diff
@@ -0,0 +1,54 @@
+Cluster Setup
+=============
+Setup Command
+###################
+This tutorial demonstrates how to set up a cluster with ``xparl`` command.
+
+To start a simple PARL cluster, we can execute the following two ``xparl`` commands:
+
+.. code-block:: bash
+
+  xparl start --port 6006
+
+  xparl connect --address localhost:6006
+
+We use the port `6006` for demonstration, and it can be any available port.
+
+**Explaination**
+
+1. start a master node to manage computation resources.
```

Comment:
Should we tell user that when we start a master, we will also start a worker in the same machine by default.

Reply:
Currently we are not adding any worker when we execute ``xparl start``..

Reply:
Done.